src/main/java/net/minecraftforge/common/capabilities/CapabilityManager.java

```diff
@@ -5,48 +5,60 @@
 
 package net.minecraftforge.common.capabilities;
 
-import net.minecraftforge.fml.ModLoader;
-import net.minecraftforge.forgespi.language.ModFileScanData;
+import net.minecraft.resources.ResourceLocation;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
 
 import java.util.IdentityHashMap;
-import java.util.List;
 
 import static net.minecraftforge.fml.Logging.CAPABILITIES;
 
+/**
+ * The Capability Manager is responsible for making and 
+ * providing {@link CapabilityType} instances.
+ */
 public enum CapabilityManager
 {
     INSTANCE;
-    static final Logger LOGGER = LogManager.getLogger();
```

Comment:
Make a decision - are you going to have a `enum` with single `INSTANCE`, and require everything to access the instance? Or are you just making everything static in which case it doesn't make any sense for this to be an `enum` in the first place.

Reply:
Enum was legacy from the prior system, though there's not much point to have it.  I'll just drop it.